platforms/nuttx/src/px4/nxp/s32k3xx/include/px4_arch/hw_description.h

```diff
@@ -131,7 +131,7 @@ enum Pin {
 struct GPIOPin {
 	Port port;
 	Pin pin;
-	uint16_t imcr;
+	uint32_t pinset;
```

Comment:
The `port` and `pin` are not used for DRDY Can we get rid them?

Reply:
Indeed DRDY can just be a `uint32_t` now.